Backpack/src/main/java/net/skyscanner/backpack/button/internal/BpkButtonBase.kt

```diff
@@ -40,13 +36,13 @@ abstract class BpkButtonBase internal constructor(
 
   abstract var iconPosition: Int
 
-  abstract var icon: Drawable?
-
   init {
     maxLines = 1
     gravity = Gravity.CENTER
     ellipsize = TextUtils.TruncateAt.END
     isClickable = isEnabled
+    iconSize = resources.getDimensionPixelSize(R.dimen.bpkSpacingBase)
```

Comment:
this automatically resizes the icon to the right side, so we no longer have to do this manually